docs/references/javascript/sign-in/sign-in.mdx

```diff
@@ -60,8 +60,8 @@ The `SignIn` object's properties can be split into logical groups, with each gro
     <li><strong><code>totp</code></strong> for TOTPs</li>
     </ul>
     As well as the identifier that the factor refers to. Please note that this property is populated only when the first factor is verified.</>]},
-    {cells: [<code>firstFactorVerification</code>, <code>Verification</code>, <>The state of the verification process for the selected first factor. Please note that this property contains an empty verification object initially, since there is no first factor selected. You need to call the <a href="/reference-js/sign-in/first-factor#preparefirstfactor"><code>prepareFirstFactor</code></a> method in order to start the verification process.</>]},
-    {cells: [<code>secondFactorVerification</code>, <code>Verification</code>, <>The state of the verification process for the selected second factor. Similar to <strong><code>firstFactorVerification</code></strong>, this property contains an empty verification object initially, since there is no second factor selected. For the <code>phone_code</code> strategy, you need to call the <a href="/reference-js/sign-in/second-factor#preparesecondfactor"><code>prepareSecondFactor</code></a> method in order to start the verification process. For the <code>totp</code> strategy you can directly attempt.</>]},
+    {cells: [<code>firstFactorVerification</code>, <code>Verification</code>, <>The state of the verification process for the selected first factor. Please note that this property contains an empty verification object initially, since there is no first factor selected. You need to call the <a href="/docs/references/javascript/sign-in/first-factor#preparefirstfactor"><code>prepareFirstFactor</code></a> method in order to start the verification process.</>]},
```

Comment:
```suggestion
    {cells: [<code>firstFactorVerification</code>, <code>Verification</code>, <>The state of the verification process for the selected first factor. Please note that this property contains an empty verification object initially, since there is no first factor selected. You need to call the <a href="/docs/references/javascript/sign-in/first-factor#prepare-first-factor"><code>prepareFirstFactor</code></a> method in order to start the verification process.</>]},
```